maestro-orchestra-models/src/main/java/maestro/orchestra/WorkspaceConfig.kt

```diff
@@ -8,17 +8,24 @@ data class WorkspaceConfig(
     val includeTags: StringList? = null,
     val excludeTags: StringList? = null,
     val local: Local? = null,
+    val executionOrder: ExecutionOrder? = null
 ) {
 
     @JsonAnySetter
     fun setOtherField(key: String, other: Any?) {
         // Do nothing
     }
 
+    // deprecated in favour of ExecutionOrder
```

Comment:
Let's add the `@Deprecated` annotation as well so it's easier to map and find in the code

Reply:
https://github.com/mobile-dev-inc/maestro/pull/1148/commits/275e0b987cc8d0deece3bb2dbecb1752015c5ec8